packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -435,3 +471,108 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     this._notifyThrottled();
   };
 }
+
+function _getStickyContainerStyle(scrollbarHeight: number, scrollbarWidth: number, isTop: boolean, height?: number): React.CSSProperties {
+  const isRtl = getRTL();
+  const left = `${scrollbarWidth || 0}px`;
+  const right = '0';
+  return {
+    ...(height !== undefined ? { height: height } : {}),
+    right: isRtl ? right : left,
+    left: isRtl ? left : right,
+    ...(isTop
+      ? {
+          top: '0'
+        }
+      : {
+          bottom: `${scrollbarHeight || 0}px`
+        })
+  };
+}
+
+function _getScrollbarWidth(contentContainer: HTMLDivElement | null): number {
+  return contentContainer ? contentContainer.offsetWidth - contentContainer.clientWidth : 0;
+}
+
+function _getScrollbarHeight(contentContainer: HTMLDivElement | null): number {
+  return contentContainer ? _getOffsetHeight(contentContainer) - contentContainer.clientHeight : 0;
+}
+
+/**
+ * Returns state.scrollbarHeight
+ * @param state Returns
+ */
+function _getStateScrollbarHeight(state: IScrollablePaneState): number {
```

Comment:
Is there a reason you have all these extra functions? I can see the reason for ones like _getScrollbarHeight which contain some calculation, but making functions to access each property on state (with no additional logic) seems like increasing the code size without any real benefit.

Reply:
If scrollbarHeight is read >=4 times, string1.length < string2. length 
[string1 ](https://microsoft-my.sharepoint.com/:u:/p/hatrived/EZYBQM9RSfxAnE_rYNFnWEoByq9BKJn8fr-iiOlKNgihQg?e=HVA4D8)= `'function yn(e){return e.scrollbarHeight}yn(e.state)yn(e.state)yn(e.state)yn(e.state)'`
vs. 
[string2 ](https://microsoft-my.sharepoint.com/:u:/p/hatrived/EQacGbgY4npCp_F3koYBlXkBFrvvPb-YARQRHOtGrz1H6g?e=cEB9xy)= `'e.state.scrollbarHeight this.state.scrollbarHeight t.scrollbarHeight n.state.scrollbarHeight'`

The idea is if name of the property is a big string & is being used frequently in the code, making functions for property access would reduce bundle size. 
Let me know if it is not fine...